frontend/views/containers/sidebar/GroupMembers.vue

```diff
@@ -97,7 +104,11 @@ export default {
     ...mapGetters([
       'groupMembers',
       'groupMembersCount'
-    ])
+    ]),
+    currentUserName () {
+      console.log('ora', this.$store.state.loggedIn.username, this.groupMembers)
```

Comment:
Why is there a comment here inside of a computed property? Before this can be merged it should be removed.

Reply:
ups, lost log to debug. removed 🙈